Modules/Filtering/DisplacementField/wrapping/itkDisplacementFieldTransform.wrap

```diff
@@ -1,5 +1,6 @@
 itk_wrap_class("itk::DisplacementFieldTransform" POINTER)
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
     itk_wrap_template("${ITKM_D}${d}" "${ITKT_D},${d}")
```

Comment:
We generally wrap all Transform type for `double` for sufficient precision in optimization. Use of the DisplacementFieldTransform results correspond to the Image[Vector] support, so wrapping should correspond -- the wrapping configuration may have `Image[Vector[float], D]` support disabled. So, we should condition this with

```
if (ITK_WRAP_vector_float)
  itk_wrap_template("${ITKM_F}${d}" "${ITKT_F},${d}")
endif()
```

Reply:
Thanks for this discussion @thewtex . I'm abandoning this PR because the functional interface `itk.transform_mesh_filter()` you suggested elsewhere is wrapped for `DisplacementFieldTransform[double,3]` already, but I'll keep precision handling in mind for future PRs.